core/src/modules/mustacheUtils.ts

```diff
@@ -47,14 +47,16 @@ export namespace MustacheUtils {
     configObjects: AnyConfigurationObject[] = []
   ) {
     const keys = getMustacheVariables(string);
-    const properties = await api.sequelize.models.Property.findAll();
+    const properties: Record<string, any>[] =
+      await api.sequelize.models.Property.findAll();
     const searchItems: Array<{ id: string; key: string }> = [].concat(
       properties.map((p) => {
         return { id: p["id"], key: p["key"] };
       }),
       configObjects
         .filter((c) => c.class.toLowerCase() === "property")
         .map((c) => {
+          //@ts-ignore
           return { id: c.id, key: c["key"] || c["name"] };
```

Comment:
```suggestion
          // @ts-ignore
          const key = c["key"] || c["name"];
          return { id: c.id, key };
```
This is a little safer imo